.github/workflows/cargo_machete.yml

```
@@ -6,7 +6,13 @@ jobs:
   cargo-machete:
     runs-on: ubuntu-latest
     steps:
+      - uses: dtolnay/rust-toolchain@master
+        with:
+          toolchain: 1.85
+      - name: Machete install
+        run: cargo install cargo-machete
```

```suggestion
        run: cargo install cargo-machete --locked
```
I think we should keep the --locked